db/rules.go

```diff
@@ -181,7 +181,13 @@ func (nh *NatsHandler) ruleRunActions(nc *natsgo.Conn, r *data.Rule, triggerNode
 			}
 		case data.PointValueActionPlayAudio:
 			channelNum := strconv.Itoa(a.PointChannel)
-			exec.Command("speaker-test", "-D"+a.PointDevice, "-twav", "-w"+a.PointFilePath, "-c5", "-s"+channelNum, "-r44100").Start()
+			go func() {
```

Comment:
@collinbrake I made a few tweaks for this so the app prints out more error info if there are audio play issues, and I removed the 44100 -- seems to play without on my PC. Do you know of any issues with this?